redis_enterprise/hatch.toml

```diff
@@ -0,0 +1,7 @@
+[env.collectors.datadog-checks]
+
+[[envs.default.matrix]]
+python = ["3.11"]
+
+[envs.default.env-vars]
+DDEV_SKIP_GENERIC_TAGS_CHECK = "true"
```

Comment:
Hmm, didn't notice this before. Why is it needed?

Reply:
because we have a label/tag/attribute called 'cluster' and we can't make it go away